SUP-2422: update deprecated keys in getContextKey

timcubb opened the pull request:
Currently getContextKey is returning "contaminations" or "confirmations", which is throwing a deprecated warning in the console, stating to use `experiments.contaminations` and `experiments.confirmations` instead.

robertsevernsentient commented:
Hey @timcubb -- can you review this PR before commting this?
https://github.com/evolv-ai/client-integrations/pull/25

Also -- can you add the same change for this file analytics/adobe-analytics/experience-accelerator/src/Client.ts?

robertsevernsentient reviewed (changes requested):
Please add the same change for analytics/adobe-analytics/experience-accelerator/src/Client.ts

timcubb commented:
> Please add the same change for analytics/adobe-analytics/experience-accelerator/src/Client.ts

@robertsevernsentient - I will rebase and add to AA now that your PR is merged.

timcubb commented:
> Hey @timcubb -- can you review this PR before commting this?
> #25
> 
> Also -- can you add the same change for this file analytics/adobe-analytics/experience-accelerator/src/Client.ts?

@robertsevernsentient - I have rebased and added the change to AA.
